src/SimpleNameRegister.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+contract SimpleNameRegister {
+    
+    // map a string to an address to identify current owner
+    mapping (string => address) public nameOwner;    
+
+    // emit event when name is registered or relinquished
+    event NameRegistered(address indexed owner, string indexed name);
+    event NameRelinquished(address indexed owner, string indexed name);
+
+    // register an available name
+    function registerName(string memory _name) public {
```

Comment:
```suggestion
    function register(string memory _name) public {
```
You could even consider `hold` as the function names, since we talk about `holders`.

Reply:
Understood. Changes made to both function name and events.